data/gui/window/statistics_dialog.cfg

```diff
@@ -176,42 +176,106 @@
 
 								[column]
 									grow_factor = 1
-									border = "all"
+									border = "left"
 									border_size = 5
 									horizontal_grow = true
 
 									[label]
-										id = "damage_type"
+										id = {colid_type}
 										definition = "default"
 										linked_group = "type"
 									[/label]
 								[/column]
 
-								[column]
-									grow_factor = 1
-									border = "all"
-									border_size = 5
-									horizontal_grow = true
-
-									[label]
-										id = "damage_overall"
-										definition = "default_small"
-										linked_group = "detail"
-									[/label]
-								[/column]
-
-								[column]
-									grow_factor = 1
-									border = "all"
-									border_size = 5
-									horizontal_grow = true
-
-									[label]
-										id = "damage_this_turn"
-										definition = "default_small"
-										linked_group = "cost"
-									[/label]
-								[/column]
+                                [column]
+                                    grow_factor = 1
+                                    border = "left"
+                                    border_size = -5
```

Comment:
Having borders around grids isn't usually done except in very specific circumstances, and negative border sizes are not supported anyway. Please remove.

Reply:
Per discussion on discord we'll address this after merge.

@Vultraz Thanks for the review! I think I've addressed everything. I've also rebased to master.

Reply:
I changed the -5 to 0 and it still looks correct. I'll merge.